ganga/GangaCore/Core/MonitoringComponent/MonitoringService.py

```diff
@@ -226,3 +226,28 @@ def stop(self):
             log.error(err)
         self._cleanup_scheduled_tasks()
         self.loop.call_soon_threadsafe(self.loop.stop)
+
+    def runMonitoring(self, jobs=None):
+        """
+        Enable/Run the monitoring loop and wait for the monitoring steps completion.
+        Parameters:
+          steps:   number of monitoring steps to run
+          timeout: how long to wait for monitor steps termination (seconds)
+          jobs: a registry slice to be monitored (None -> all jobs)
+        Return:
+          False, if the loop cannot be started or the timeout occured while waiting for monitoring termination
+          True, if the monitoring steps were successfully executed  
+        Note:         
+          This method is meant to be used in Ganga scripts to request monitoring on demand. 
+        """
+
+        log.debug("runMonitoring")
+        if not self.alive:
+            log.error("Cannot run the monitoring loop. It has already been stopped")
+            return False
+
+        self.enabled = True
```

Comment:
I am not wholly enthused by this. You have to set `enabled` to `True` to monitor the jobs but in reality it is not monitoring everything.